include_defines/ideal_MHD_defines.h

```diff
@@ -57,6 +57,9 @@ TODO: JANHUNEN_CENTRAL, BRACKBILL_BARNES_CENTRAL, cf.
 */
 #define USE_SOURCE_GODUNOV_CENTRAL
 
+// Speeds: USE_VR_KUSANO USE_VR_BKW
+#define USE_VR_KUSANO
```

Comment:
What does `VR` mean?

Since we might want to add some additional boundary procedures in the future, I would propose to use something along the lines of 
```C
#define USE_BOUNDARY_FLUX_HLL
#define USE_HLL_SPEEDS_<WHATEVER>
```

Reply:
i changed the names. We should be aware of the fact that USE_BOUNDARY_FLUX_HLL is the opposite of USE_PERIODIC.

Reply:
I had a slightly different structure in mind: `USE_BOUNDARY_FLUX_HLL` can be defined independently of `USE_PERIODIC`. If `USE_PERIODIC`, we must specify some boundary flux, but that could be `USE_FLUX_YourfavoriteFluxThatWillBeImplemented` in the future. I thought of something along the lines of
```C
#ifndef UES_PERIODIC

#if defined USE_BOUNDARY_FLUX_HLL

// the HLL stuff, including different speeds

// might be implemented in the future
//#elif defined USE_BOUNDARY_FLUX_LLF

#else 

#error "Error in include_physics/ideal_MHD.h: Non-periodic boundaries are used but no boundary flux is specified!"

#endif // USE_BOUNDARY_FLUX_
```

Reply:
Sorry. I misunderstood your comment.

Reply:
No problem.